### ISSUE TITLE
Provide uninstall script

### ISSUE THREAD

**Issue by cnSchwarzer** [DamionGans/ubuntu-wsl2-systemd-script#26]
Thank you.

**Comment by nsi-test** [comment]
I totally agree. It turned out that it is ok, but actually I don’t need the systemd. Sometimes I have problems on service startup via wsl, the script “service” works, anyway.

**Comment by HotPotatoC** [comment]
Almost a year late but, A [fork](https://github.com/FiestaLake/ubuntu-wsl2-systemd-script) of this repo has an [uninstall script](https://github.com/FiestaLake/ubuntu-wsl2-systemd-script/blob/master/uninstall.sh) and it works as expected.

**Comment by yantom** [comment]
Actually for me just in time! I used the original script yesterday and then I used the provided uninstall script. Both have worked without problem.
